### PR TITLE
feat(auth): Added VerifySessionCookieAsync() APIs

### PR DESCRIPTION
**Pull request by hiranya911**
```
Task<FirebaseToken> VerifySessionCookieAsync(string cookie)
Task<FirebaseToken> VerifySessionCookieAsync(string cookie, boolean checkRevoked)
```

RELEASE NOTE: Added `VerifySessionCookieAsync()` method for verifying Firebase session cookies in backend applications.

### CODE REVIEW

**Comment by hiranya911** [comment]
@kevinthecheung please review the API reference docs in `FirebaseAuth` class.

File: FirebaseAdmin/FirebaseAdmin/Auth/FirebaseAuth.cs [hunk 1]
```diff
@@ -844,6 +847,113 @@ public async Task<string> GeneratePasswordResetLinkAsync(string email)
...
+        /// Parses and verifies a Firebase session cookie.
+        ///
+        /// <para>See <a href="https://firebase.google.com/docs/auth/admin/manage-cookies">Manage
+        /// Session Cookies</a> for code samples and detailed documentation.</para>
+        /// </summary>
+        /// <returns>A task that completes with a <see cref="FirebaseToken"/> representing
+        /// the verified and decoded session cookie.</returns>
+        /// <exception cref="ArgumentException">If the session cookie is null or
+        /// empty.</exception>
+        /// <exception cref="FirebaseAuthException">If the session cookie fails to
```
Comment (kevinthecheung) [inline FirebaseAdmin/FirebaseAdmin/Auth/FirebaseAuth.cs:860]: fails to verify -> is invalid

File: FirebaseAdmin/FirebaseAdmin/Auth/FirebaseAuth.cs [hunk 2]
```diff
@@ -844,6 +847,113 @@ public async Task<string> GeneratePasswordResetLinkAsync(string email)
...
```
Comment (kevinthecheung) [inline FirebaseAdmin/FirebaseAdmin/Auth/FirebaseAuth.cs:880]: fails to verify -> is invalid

File: FirebaseAdmin/FirebaseAdmin/Auth/FirebaseAuth.cs [hunk 3]
```diff
@@ -844,6 +847,113 @@ public async Task<string> GeneratePasswordResetLinkAsync(string email)
...
```
Comment (kevinthecheung) [inline FirebaseAdmin/FirebaseAdmin/Auth/FirebaseAuth.cs:903]: fails to verify -> is invalid

File: FirebaseAdmin/FirebaseAdmin/Auth/FirebaseAuth.cs [hunk 4]
```diff
@@ -844,6 +847,113 @@ public async Task<string> GeneratePasswordResetLinkAsync(string email)
...
```
Comment (kevinthecheung) [inline FirebaseAdmin/FirebaseAdmin/Auth/FirebaseAuth.cs:927]: fails to verify -> is invalid

Comment (hiranya911) [reply FirebaseAdmin/FirebaseAdmin/Auth/FirebaseAuth.cs]: Done

File: FirebaseAdmin/FirebaseAdmin/Auth/FirebaseAuth.cs (hunk 3 above)
Comment (hiranya911) [reply FirebaseAdmin/FirebaseAdmin/Auth/FirebaseAuth.cs]: Done

File: FirebaseAdmin/FirebaseAdmin/Auth/FirebaseAuth.cs (hunk 2 above)
Comment (hiranya911) [reply FirebaseAdmin/FirebaseAdmin/Auth/FirebaseAuth.cs]: Done

File: FirebaseAdmin/FirebaseAdmin/Auth/FirebaseAuth.cs (hunk 1 above)
Comment (hiranya911) [reply FirebaseAdmin/FirebaseAdmin/Auth/FirebaseAuth.cs]: Done

**Comment by hiranya911** [comment]
Thanks @kevinthecheung. I've made the suggested changes.

**Comment by hiranya911** [comment]
Resolves #160

**Pull request merged**